docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,71 @@
+## Checklist
+<!--
+Mark an `[x]` for completed items.
+-->
+
+* [ ] Update CHANGELOG.rst for each commit separately:
+  * Pay attention to write entries under the "Unreleased" section.
+  * Mark all breaking changes as "**BACKWARD INCOMPATIBLE:**" and put them
+    before non-breaking changes.
+  * If a commit modifies a feature listed under "Unreleased" section,
+    it might be sufficient to modify the existing CHANGELOG entry from previous
+    commit(s). When making a new CHANGELOG entry, augment it.
+* [ ] Bump the process version:
+  * **MAJOR version (first number)**: Backwards incompatible changes (changes
+    that break the api/interface). Examples: renaming the input/output, adding
+    mandatory input, removing input/output…
+  * **MINOR version (middle number)**: add functionality or changes in a
+    backwards-compatible manner. Examples: add output field, add non-mandatory
+    input parameter, use a different tool that produces same results...
+  * **PATCH version (last number)**: changes/bug fixes that do not affect
+    the api/interface. Examples: typo fix, change/add warning messages...
+* [ ] All inputs are used in process.
+* [ ] All output fields have their ``re-save``.
+
```

Comment:
Add a comment indicating that the following lines should be read and discarded before opening a PR.

Something like:
```
<!--
Read the following guidelines and remove them before opening the pull request.
-->
```